internal/cli/command/cluster/integratedservice/services/backup/enable.go

```diff
@@ -223,19 +223,35 @@ func buildEnableRequestInteractively(banzaiCli cli.Cli, options enableOptions) (
 		return pipeline.EnableArkRequest{}, errors.WrapIf(err, "failed to get bucket")
 	}
 
+	serviceAccountRoleARN := ""
 	useClusterSecret := false
 	if options.ClusterCloud() == amazonType {
 		err = input.DoQuestions([]input.QuestionMaker{
-			input.QuestionConfirm{
+			input.QuestionInput{
+				QuestionBase: input.QuestionBase{
+					Message: "Service Account Role ARN to use for Velero",
+					Help:    "This IAM role ARN will be used to annotate service account for Velero and should be associated with relevant snapshot permissions."},
```

Comment:
**Recommended**: we **SHOULD** highlight the optional nature of the service account role ARN for the users to convey the possible usages more clearly.

```suggestion
					Help:    "This optional IAM role ARN will be used to annotate the service account for Velero and should be associated with the relevant snapshot permissions. Alternatively, when no service account role ARN is specified, either an optional cluster secret can be deployed or a node instance role can be extended with the snapshot permissions for creating volume snapshots with Velero."},
```